resources/contenttweaker/lang/zh_cn.lang

```diff
@@ -589,6 +589,12 @@ tooltips.dim_stages.restricted=§c在§b空岛§c世界中，你无法前往任
 tooltips.dim_stages.enter_skyblock=§e你进入了一个§b空岛§e世界。你将无法前往任何其他维度§8(不包括太空、RF工具的维度和个人维度)。另外你现在会获得持续 3 小时的急迫 IV 效果。§r
 tooltips.dim_stages.remind_skyblock=§e你再次进入了一个§b空岛§e世界。谨记：你无法前往任何其他维度§8(不包括太空、RF工具的维度和个人维度)§r。
 
+# --------------------------------------
+# Custom JEI categories
+# - "e2ee.jei.{id}.{entry}" format recommended
+# --------------------------------------
+e2ee.jei.rats_consuming.title=老鼠食用
```

Comment:
Please add `ru_ru.lang` file string - amount of lines in each .lang file should be the same. Translate them with AI or deepl.com